.github/release-please-config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "release-type": "node",
+  "separate-pull-requests": true,
+  "include-component-in-tag": true,
```

Comment:
q: I see `include-component-in-tag` is the equivalent "new option" to the old `monorepo-tags`, but we didn't have `separate-pull-requests` enabled in our previous config, what does it do?

Reply:
It controls whether the action should create separate PRs for each release step (e.g., draft release, release candidate, final release) or combine all changes into a single pull request. This option is set to true on `w3up`, which is why I included it here. But now that you mention it, I think we may not need all of this.